pages/upgrading_an_app.md

```diff
@@ -62,6 +62,21 @@ If you are doing the upgrade more than once you could consider to first upgrade
 
 With doing the above you upgrade the jhipster_upgrade tree with your latest changes so JHipster can make use of that during the upgrade. For example when you changed your model.
 
+### Upgrading from JHipster 7 and previous versions
+
+The upgrade sub-generator is a light version of the [migrate blueprint](https://github.com/jhipster/generator-jhipster-migrate). If you need to upgrade a JHipster 7 app, it's recommended you use the migrate blueprint. For example, let's assume you have a 7.9.3 app.
```

Comment:
@mshima Is it correct to say the upgrade sub-generator is a light version of the migrate blueprint?

Reply:
I would say that migrate is an advanced version of upgrade instead.

Reply:
Done.